src/YQDialog.cc

```diff
@@ -805,6 +805,8 @@ YQDialog::waitForEventInternal( int timeout_millisec )
 YEvent *
 YQDialog::pollEventInternal()
 {
+    _eventLoop->wakeUp();
```

Comment:
To me it looks a bit strange without any comment, many questions like why this call? why here? WTF? just pop up when seeing this...

Reply:
@shundhammer do you know some background info for this call which we could add as a comment?

I'm fine with *"call this here because it works elsewhere"* solution but if possible we should know some more details just in case it breaks again in the future...

Reply:
Well, `_eventLoop->wakeUp()` **is** used in the (very) related `YQDialog::aitForEventInternal()` method just above this method, so now the sequence of Qt calls is very much the same in both those methods.

But why didn't we need this in the past? Something must have changed internally deep inside Qt. So far, I could not find any relevant entry in the Qt change logs. But I'll keep looking.